docs/reference/configuration.md

```diff
@@ -34,6 +34,7 @@ This reference uses `.` to denote the nesting of values.
 * `database.max_idle_connections` `(int : 25)` - Sets the maximum number of connections in the idle connection pool
 * `database.connection_max_lifetime` `(duration : 5m)` - Sets the maximum amount of time a connection may be reused
 * `listen_address` `(string : "0.0.0.0:8000")` - A `<host>:<port>` structured string representing the address to listen on
+* `actions_disabled` `(bool : false)` - Setting this to true will block hooks from being executed
```

Comment:
`actions.enabled` perhaps?

1. Positives are easier on my mind than double negatives
2. Hierarchy allows us to add more properties in future.
3. Matches `cache.enabled`, below.

Reply:
Done.